examples/client/client.go

```diff
@@ -0,0 +1,28 @@
+// +build go1.11
+
+package main
+
+import (
+	"fmt"
+	"github.com/valyala/fasthttp"
+	"github.com/valyala/fasthttp/reuseport"
```

Comment:
Please put the above to imports in a separate block like everywhere else ([for example](https://github.com/valyala/fasthttp/blob/master/compress.go)).